bugfix: primary navigation (#79)
clhenrick/clhenrick.io: content/about/talks/index.md

---
layout: layouts/about.njk
title: Talks and Workshops

> description: "A list of lectures, presentations, and workshops given by Chris Henrick, ordered by date."

Author: ```suggestion
description: "Lectures, presentations, and workshops in reverse chronological order."
```
